src/etc/inc/filter_log.inc

```diff
@@ -79,10 +79,60 @@ function conv_log_filter($logfile, $nentries, $tail = 50, $filtertext = "", $fil
 	/* Always do a reverse tail, to be sure we're grabbing the 'end' of the log. */
 	$logarr = "";
 
+	    if ($logfile == "{$g['varlog_path']}/system.log")		{ $logfile_type = "system"; }
+	elseif ($logfile == "{$g['varlog_path']}/gateways.log")		{ $logfile_type = "system"; }
```

Comment:
I should check the style guide, but every other file I have worked on uses "else if" (with a space).